src/IW/Core/Issue.hs

```diff
@@ -5,25 +5,20 @@ module IW.Core.Issue
        , issueUrl
        ) where
   
-import IW.Core.Id (Id (..))
 import IW.Core.SqlArray (SqlArray (..))
 import IW.Core.Repo (RepoName (..), RepoOwner (..))
 
 
 -- | Data type representing a GitHub issue.
 data Issue = Issue
-    { issueId        :: Id Issue
-    , issueRepoOwner :: RepoOwner
+    { issueRepoOwner :: RepoOwner
     , issueRepoName  :: RepoName
     , issueNumber    :: Int
     , issueTitle     :: Text
```

Comment:
While we are on this: let's make every field in our `Repo` and `Issue` data types strict.